ragstack/colbert/cassandra_db.py

```diff
@@ -0,0 +1,183 @@
+from typing import List
+from cassandra.cluster import Cluster
+from cassandra.auth import PlainTextAuthProvider
+from cassandra import InvalidRequest
+from cassandra.concurrent import execute_concurrent_with_args
+
+from .token_embedding import PassageEmbeddings
+
+
+def required_cred(cred: str):
+    if cred is None or cred == "":
+        raise ValueError("Please provide credentials")
+
+
+class AstraDB:
+    def __init__(
+        self,
+        secure_connect_bundle: str = "",
+        astra_token: str = None,
+        keyspace: str = "colbert128",
+        verbose: bool = False,
+        timeout: int = 60,
+        **kwargs,
+    ):
+        required_cred(secure_connect_bundle)
+        required_cred(astra_token)
+
+        # self.cluster = Cluster(**kwargs)
+        self.cluster = Cluster(
```

Comment:
still no cassio ? 
I think you can just assume here that cassio.init has been called and get the session by using

`session = cassio.resolve_session()`

Reply:
The integration test uses a Cassandra container now.